src/index.tsx

```diff
@@ -1,8 +1,64 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
+import "./styles.css";
+import { ApolloClient } from "apollo-client";
+import { ApolloProvider } from "@apollo/react-hooks";
+import { GraphQLSchema } from "graphql/type";
+import { HttpLink } from "apollo-link-http";
+import { InMemoryCache, NormalizedCacheObject } from "apollo-cache-inmemory";
+import { introspectSchema, makeExecutableSchema } from "graphql-tools";
+import { isDevelopment, isTest, getSchemaString, getGQLUrl } from "./utils";
+import { printSchema } from "graphql/utilities/schemaPrinter";
+import { SchemaLink } from "apollo-link-schema";
 import Evergreen from "./components/app/App";
 import registerServiceWorker from "./registerServiceWorker";
-import "./styles.css";
+import resolvers from "./gql/resolvers";
+
+async function getClientLink(): Promise<HttpLink | SchemaLink | null> {
+  const gqlURI: string = getGQLUrl();
```

Comment:
you can avoid defining types in situations like this where it's inferred from the function